go/private/rules/transition.bzl

```diff
@@ -224,6 +168,46 @@ def _go_transition_impl(settings, attr):
 
     return settings
 
+def _non_go_transition_impl(settings, attr):
```

Comment:
Not: Could you clean up the combined diff by moving this definition back to it's original place?

Reply:
Done!